NickvisionTagger.Shared/Models/MusicFile.cs

```diff
@@ -854,23 +854,31 @@ public override bool Equals(object? obj)
     /// <returns>True if a < b, else false</returns>
     public static bool operator <(MusicFile? a, MusicFile? b)
     {
+        if(SortFilesBy == SortBy.Path)
```

Comment:
Use a `switch` statement

Reply:
Using a switch wouldn't solve anything...it'd make it more messy adding case and then break everywhere.

Reply:
Then use the switch expression